### PR TITLE
Add button platform to Motionblinds BLE

### PR DESCRIPTION
**Pull request by LennP**
## Proposed change

Add button platform to Motionblinds BLE

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/114227
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/motionblinds_ble.markdown [hunk 1]
```diff
@@ -11,6 +11,7 @@ ha_codeowners:
   - '@jerrybboy'
 ha_config_flow: true
 ha_platforms:
+  - button
```
Comment (c0ffeeca7) [inline source/_integrations/motionblinds_ble.markdown:14]: Could add a description of the button functionality that was added? 
Something like this, for inspiration:
https://github.com/home-assistant/home-assistant.io/blob/7b9db946866a38e591b439343c0d23017f5c14df/source/_integrations/plex.markdown?plain=1#L105-L141

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/motionblinds_ble.markdown (hunk 1 above)
Comment (LennP) [reply source/_integrations/motionblinds_ble.markdown:14]: > Could add a description of the button functionality that was added? Something like this, for inspiration:
> 
> https://github.com/home-assistant/home-assistant.io/blob/7b9db946866a38e591b439343c0d23017f5c14df/source/_integrations/plex.markdown?plain=1#L105-L141

Sure fixed it the same way I did for select and sensor.

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @LennP :+1:

**Review by edenhaus — DISMISSED** [review]
Please fix the merge requests, afterwards we can merge this one too :)

**Comment by LennP** [comment]
> Please fix the merge requests, afterwards we can merge this one too :)

Fixed it, thanks!

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**